examples/fingerprint_picturetest.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
I would change this to your name, as well as remove the credit to Adafruit Industries:

```suggestion
# SPDX-FileCopyrightText: 2023 Jim McKeown
```